doc/classes/TextureProgress.xml

```diff
@@ -52,10 +52,13 @@
 			[Texture] that draws under the progress bar. The bar's background.
 		</member>
 		<member name="tint_over" type="Color" setter="set_tint_over" getter="get_tint_over">
+			Multiplies the color of the bar's [code]texture_over[/code] texture. The effect is similar to [member CanvasItem.modulate], except it only affects this specific texture instead of the entire node.
 		</member>
 		<member name="tint_progress" type="Color" setter="set_tint_progress" getter="get_tint_progress">
+			Multiplies the color of the bar's [code]texture_progress[/code] texture.
 		</member>
 		<member name="tint_under" type="Color" setter="set_tint_under" getter="get_tint_under">
+			Multiplies the color of the bar's [code]texture_under[/code] texture.
```

Comment:
This sentence (as well as the first sentence of the other two), while correct from what I can tell, seems to me like it might be a little bit confusing. Perhaps something like `Tints the color of the [code]texture_under[/code] texture.` or something of the sort?

Reply:
It doesn't tint the texture: it multiplies it, darkening it along the way, like Canvasitem.modulate. It's applying a color with the multiply blending mode artists and designers are very familiar with.

Reply:
Ah... then why is it called `tint_*`? That seems a little confusing.

Either way, if that's the case, then feel free to mark this as resolved.

Reply:
For the designer who looks at the properties in the inspector, tint is short... I guess modulate_* would've made sense as it's the name CanvasItem uses already.
Yeah sure, I test every property and read the source code if need be when I write docs